src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+- The hidden charge of financial services is your personal data
+- Governments can close down markets at will
+- Money transfers can take days due to internal human processes
+- There's a premium to financial services because intermediray institutions need their cut
+
+| Defi                        | Traditional finance                                      |
+| --------------------------- | -------------------------------------------------------- |
+| You control your funds      | Your funds are looked after by companies                 |
+| Transfers happen in minutes | Payments can take days due to manual processes           |
+| Pseudonymous                | Financial activity is tightly coupled with your identity |
+| Open to anyone              | You must apply to use financial services                 |
+
+<ButtonLink to="/dapps/?category=finance">Explore defi apps</ButtonLink>
+
+## What can you do with defi?
+
+There's a decentralized alternative to most financial services. But Ethereum also creates opportunities for creating financial products that are completely new. This is an ever-growing list.
+
+- [Send money around the globe](#send-money)
+- [Stream money around the globe](#stream-money)
+- [Access stable currencies](#stablecoins)
+- [Lend/borrow funds](#lending)
+- [Borrow without collateral](#flash-loans)
+- [Start crypto savings](#savings)
+- [Swap tokens](#swaps)
+- [Trade??](#trading)
+- [Grow your portfolio](#investing)
+- [Fund your ideas](#crowdfunding)
+
+### Send money around the globe quickly {#send-money}
+
+As a blockchain, Ethereum is designed for sending transactions in a secure and global way. Payments go directly from one account to another in a matter of minutes (usually). To send/receive payments you'll need a wallet.
+
+<DocLink to="/wallets/" title="More on wallets" />
+
+#### Stream money around the globe... {#stream-money}
+
+It's also possible to stream money over Ethereum. This would let you pay someone their salary by the second, giving them access to their money whenever they need it.
+
+### Access to stable currencies {#stablecoins}
+
+Although a lot of cryptocurrencies are known for their volatility, the defi community has invested a lot of time in creating stability out of volatility with stablecoins. Their value stays pegged to an another asset, usually a popular currency like dollars.
+
+Coins like Dai or USDC have a value that stays within a few cents of a dollar. This makes them perfect for earning or retail. So far many people in Latin America have used stablecoins as a way of protecting their savings in a time of great uncertainty with their government-issued currencies.
+
+<DocLink to="/stablecoins/" title="More on stablecoins" />
+
+### Lending and borrowing {#lending}
+
+Today, lending and borrowing money all revolves around the individuals involved. Banks need to know whether you're likely to repay a loan before lending. Some see this as an invasion of privacy.
```

Comment:
I wonder what the biggest value prop of decentralized borrowing is. Is it the fact that your privacy is protected? I think the argument here from critics would be "well you don't have to run a credit report because you're providing collateral - you could do the same at a pawn shop". The reason banks check your credit history is because those loans aren't typically collateralized.

My guess is one main value prop is the global liquidity this allows: connecting borrows & lenders from across the globe. In theory, this gives greater access to loans, which equals better interest rates.

Lending value prop is similar - access to demand from around the globe. This gives optionality vs. your bank, which provides extremely low interest rates.

The other obvious value prop for borrowers is being able to borrow money without exiting your position (e.g. selling your ETH & facing a tax event) but not sure that's worth getting into here 😄

Reply:
I think the tax note is actually really important. People who want to hold, but need money, often neglect this option and sell. This creates a potential tax burden for gains, so you're immediately paying basically income tax on the amount you sold, and now you don't own the asset anymore, and if it does go up in value, you lost your exposure. A loan provides cash flow, keeps you in your position, and in some cases paying off the interest could be a write-off.

Reply:
Nice.  have expanded this section. Just to confirm, borrowing interest rates are better because of a larger global liquidity pool?